go/pkg/cas/client.go

```diff
@@ -145,9 +158,30 @@ func (c *ClientConfig) Validate() error {
 		return fmt.Errorf("FileIOSize must be positive")
 
 	// Do not allow more than 100K, otherwise we might run into the request size limits.
-	case c.FindMissingBlobsBatchSize <= 0 || c.FindMissingBlobsBatchSize > 10000:
-		return fmt.Errorf("FindMissingBlobsBatchSize must be in [1, 10000]")
+	case c.FindMissingBlobs.MaxItems > 10000:
+		return fmt.Errorf("FindMissingBlobs.MaxItems must <= 10000")
+	}
 
+	if err := c.FindMissingBlobs.validate(false); err != nil {
+		return errors.Wrap(err, "FindMissingBlobs")
+	}
+	if err := c.BatchUpdateBlobs.validate(true); err != nil {
+		return errors.Wrap(err, "BatchUpdateBlobs")
+	}
+	return nil
+}
+
+// validate returns an error if the config is invalid.
+func (c *RPCConfig) validate(validateMaxSizeBytes bool) error {
```

Comment:
I'm not sure if validating MaxSizeBytes makes sense. If we're validating that, we might as well validate MaxItems for streams... And adding one option per RPC sounds complex.

Reply:
removed.